airbyte-webapp/src/views/layout/SideBar/components/SidebarDropdownMenu.tsx

```diff
@@ -0,0 +1,68 @@
+import { Menu } from "@headlessui/react";
+import classNames from "classnames";
+import React from "react";
+
+import styles from "./SidebarDropdownMenu.module.scss";
+
+export enum SidebarDropdownMenuItemType {
+  LINK = "link",
+  BUTTON = "button",
+}
+
+interface MenuItemLink {
+  type: SidebarDropdownMenuItemType.LINK;
+  href: string;
+  icon: React.ReactNode;
+  displayName: React.ReactNode;
+}
+
+interface MenuItemButton {
+  type: SidebarDropdownMenuItemType.BUTTON;
+  icon: React.ReactNode;
+  displayName: React.ReactNode;
+  onClick: () => void;
+}
+
+export const SidebarDropdownMenu: React.FC<{
+  children: React.ReactNode;
```

Comment:
There is no need to specify `children` manually in properties. `React.FC` does automatically add this to the properties you pass in.

Reply:
Well, it depends...
If it's a component that requires children to be rendered, we can specify "children" as a required prop.
This way we will help TypeScript to give us a hint to provide children 😊